client/post-editor/media-modal/gallery/preview.jsx

```diff
@@ -72,34 +72,17 @@ export default React.createClass( {
 			);
 		}
 
-		if ( settings && 'individual' === settings.type ) {
+		if ( 'individual' === settings.type ) {
```

Comment:
>Did you remove the check for `settings` since it has a default Prop here?

Yeah, and because we call a property on it earlier in the function which would blow up if it didn't exist anyways, so the check here was a bit confusing:

https://github.com/Automattic/wp-calypso/blob/acb3c66587d01ea74bf26fa56202bc7b7d067fd3/client/post-editor/media-modal/gallery/preview.jsx#L62